### ISSUE TITLE
use full-color PK icon when highlighted in blue

### ISSUE THREAD

**Issue by tomholub** [FlowCrypt/flowcrypt-browser#1951]
It's good, better than before. I think, for the blue highlight, we should be using a "filled up icon" version of the icon, in blue. That would make it more obvious. I'll make that another issue, for sometime later.

_Originally posted by @tomholub in https://github.com/FlowCrypt/flowcrypt-browser/pull/1949#issuecomment-521674333_

**Comment by tomholub** [comment]
Something like this:

![image](https://user-images.githubusercontent.com/6306961/63104736-0ce32a80-bf6f-11e9-84f9-7b17bee04bf9.png)

![image](https://user-images.githubusercontent.com/6306961/63104742-0f458480-bf6f-11e9-9a7d-46812c3fdbc5.png)

Which is improvement over:

![image](https://user-images.githubusercontent.com/6306961/63104768-1c627380-bf6f-11e9-8b13-e9593d54de90.png)

The intended blue color is: `#4d90fe`

**Comment by tomholub** [comment]
Setting this as later: after #2028 and #2039 there will be almost no icons to highlight. However, maybe the PK icon could then get moved next to send (where sign icon is now), like this:

![image](https://user-images.githubusercontent.com/6306961/65749867-8f2a4700-e0f6-11e9-91fc-445608398e5d.png)

**Comment by tomholub** [comment]
low impact, I'll close it
